Plugins/Wox.Plugin.Program/Wox.Plugin.Program.csproj

```diff
@@ -150,13 +158,6 @@
     </ProjectReference>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="..\..\packages\UwpDesktop.10.0.14393.3\build\portable-net45+uap\UwpDesktop.targets" Condition="Exists('..\..\packages\UwpDesktop.10.0.14393.3\build\portable-net45+uap\UwpDesktop.targets')" />
```

Comment:
Please reverse the UWPDesktop Nuget package removable. This is required for Program plugin to work properly. May consider another option in the future as discussed in some PRs in the upstream repo.